inspirehep/modules/authors/forms.py

```diff
@@ -563,6 +563,12 @@ def __init__(self, *args, **kwargs):
         """Constructor."""
         super(AuthorUpdateForm, self).__init__(*args, **kwargs)
         is_review = kwargs.pop('is_review', False)
+        is_update = kwargs.pop('is_update', False)
+
+        if is_update:
+            # remove validation for duplicate ORCIDs on update
+            self.orcid.validators = self.orcid.validators[:-1]
```

Comment:
Can we do better than this? Something like a sanity check that the ORCID corresponds to what was there before, or that the ORCID profile has a name that is similar enough or...

Reply:
we could do lots of things. Currently, we have a major regression: any author update fails if the author already has an ORCID. Let's fix that first.